### PR TITLE
Link for volunteer form added

### PR DESCRIPTION
**Pull request by krishamehta**

### CODE REVIEW

File: faqs.html [hunk 1]
```diff
@@ -146,7 +146,7 @@
 											<h4><span>Q.</span>I want to volunteer for the conference. What should I do?</h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>Please share your details by filling this form and the team will soon get back to you with responsibilities according to the domains you choose.</p>
+													<p><span>A.</span>Please share your details by filling this <a href = "https://docs.google.com/forms/d/1WXkev5oYAw9KiW0H6viIB7c3S-wRkFRI8BCCcctdkLk">form</a> and the team will soon get back to you with responsibilities according to the domains you choose.</p>
```
Comment (CuriousLearner) [inline faqs.html:149]: The link to the form should be this: http://bit.ly/inpycon2017

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Apart from the review mentioned inline, the changes should be made in templates file and reflected HTML pages should be added in the PR. Please read the `README.md` on how to do it.

**Comment by CuriousLearner** [comment]
Hey @krishamehta 

Just wanted to ping you to know the status of this. Will you be able to make changes to the PR?

**Comment by krishamehta** [comment]
Yes I'll do it by today

**Comment by CuriousLearner** [comment]
Thanks for the update!

**Comment by CuriousLearner** [comment]
Hey @krishamehta

Would you be able to pick this up, or someone else should go with it?

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Also, please do the changes in `templates/faqs.html` and then regenerate the HTML

File: templates/faqs.html [hunk 2]
```diff
@@ -59,7 +59,7 @@
 											<h4><span>Q.</span>I want to volunteer for the conference. What should I do?</h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>Please share your details by filling this form and the team will soon get back to you with responsibilities according to the domains you choose.</p>
+													<p><span>A.</span>Please share your details by filling this <a href = " http://bit.ly/inpycon2017">form</a> and the team will soon get back to you with responsibilities according to the domains you choose.</p>
```
Comment (CuriousLearner) [inline templates/faqs.html:62]: Please do not leave space in the hyperlink reference tag.

`<a href="http://bit.ly/inpycon2017">form</a>`

Also, please generate the HTML and it in this PR.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]

File: templates/faqs.html (hunk 2 above)
Comment (CuriousLearner) [reply templates/faqs.html:62]: Please resolve conflicts with the base master branch.

If possible, do a rebase.

Comment (krishamehta) [reply templates/faqs.html:62]: ya ya working on it

**Comment by CuriousLearner** [comment]
Due to no response on PR from a long time, this is resolved in 1a470b681f76c68e8f959d8fc32cab383639c6dc

**Pull request closed without merging**